versions/wmde3-alpha.env

```diff
@@ -0,0 +1,24 @@
+WMDE_RELEASE_VERSION=wmde.3
+WIKIBASE_BRANCH_NAME=REL1_36
+MEDIAWIKI_BRANCH_NAME=REL1_36
+
+WDQS_VERSION=0.3.40
+WDQS_FRONTEND_COMMIT_HASH=d33a20e6e232e35cb5400df97405f6ec53758389
+
+QUICKSTATEMENTS_COMMIT_HASH=757d238cf6d306f9daf8276f620415cf09f4afe6
+MAGNUSTOOLS_COMMIT_HASH=81602412a886757ace2b69f209be454b26de90ad
+WIKIBASELOCALMEDIA_COMMIT_HASH=813418de797ac63778770c0bc20994b62fc8d610
```

Comment:
Shouldn’t there be something like a WIKIBASEEDTF_COMMIT_HASH here?

Reply:
Yep, you are right